Package@swift-5.5.swift

```diff
@@ -0,0 +1,56 @@
+// swift-tools-version:5.5
```

Comment:
This remains here now for Xcode 13/14 compatibility. Moving forward, the main `Package.swift` is what Xcode 15 will use.